foundation/special.css

```diff
@@ -0,0 +1,82 @@
+body {
+    font-family: Arial,Helvetica,Verdana,sans-serif;
+    font-size: 15px;
+    line-height: 22px;
+    color: #666666;
+}
+
+h1 {
+    font-size: 32px;
+    line-height: 36px;
+    color: #333333;
+    margin-bottom: 18px;
+}
+
+h2 {
+    font-size: 24px;
+    line-height: 22px;
+    color: #333333;
+    margin-bottom: 14px;
+}
+
+h3 {
+    font-size: 18px;
+    line-height: 22px;
+    color: #333333;
+    margin-bottom: 6px;
+}
+
+p {
+	  font-size: 15px;
+    line-height: 22px;
+    margin-bottom: 22px;
+}
+
+small {
+	  font-size: 13px;
+    line-height: 18px;
+    margin-bottom: 3px;
+}
+
+/* SPACES */
+
+.margin--0 {
```

Comment:
double dash?

Reply:
This is a recommendation? I didn't write this with double dash =(